daemon/graphdriver/windows/windows_test.go

```diff
@@ -0,0 +1,20 @@
+//+build windows
+
```

Comment:
This is a test file, so it must end with _test.go. To get rid of the build tag, apparently I need to call it windows_windows_test.go, or <somethingelse>_windows_test.go.

Really we should probably rename windows.go to driver_windows.go, and this could be driver_windows_test.go. But that seems like a separate change.

What would you prefer?